src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -57,4 +57,18 @@ public boolean isValid() {
      * e.g. trim strings
      */
     public abstract void sanitizeForSaving();
+
+    /**
+     * Adds {@code error} to {@code errors} if {@code error} is a non-empty string.
+     *
+     * @param error An {@code error} message, possibly empty.
```

Comment:
This is the wrong use of `{@code ...}`. This `error` does not refer to the parameter named `error`.